### PR TITLE
feat: add context for unconditional safety

### PR DESCRIPTION
**Pull request by 0xkydo**
modified language around unconditional economic safety and context for it wrt the StakeSure paper.

### CODE REVIEW

**Review by wesfloyd — DISMISSED** [review]
looks good, approved

File: docs/risk/risk-faq.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ Moreover, another class of protocols built on top of EigenLayer is called liquid
...
 
-In addition to pooled security, EigenLayer will let each AVS acquire additional attributable security. Unlike pooled security, attributable security is specific to each AVS and will only be slashable by one AVS. This arrangement can provide game-theoretic guarantees for AVS customers. 
+In addition to pooled security, EigenLayer will let each AVS acquire additional attributable security. Unlike pooled security, attributable security is specific to each AVS and will only be slashable by one AVS. Under this design, the end goal is to provide unconditional economic safety (as defined in the StakeSure paper) for AVS end consumers. Using a bridge AVS as an example, it has a shared security base of \$10B and an attributable portion of \$10B. If the shared security part is slashed, the AVS can assure the bridge customer that up to \$10B can still be securing the bridge through the attributable part.
```
Comment (wadealexc) [inline docs/risk/risk-faq.md:19]: Please italicize _unconditional economic safety_ (or wrap in quotes) so it's more clear it's an outside concept

And please link the StakeSure paper

**Review by wadealexc — CHANGES_REQUESTED** [review]
Requesting minor changes, otherwise looks fine

File: docs/risk/risk-faq.md (hunk 1 above)
Comment (0xkydo) [reply docs/risk/risk-faq.md]: added the change here: https://github.com/Layr-Labs/eigenlayer-docs/pull/62/commits/6604b61678ea5a1f7f4eeec82f36a1560473127f

**Review by wadealexc — APPROVED** [review]
LGTM

**Pull request closed without merging**